packages/odyssey-react/src/components/Infobox/Infobox.tsx

```diff
@@ -99,7 +98,7 @@ export const Infobox = withTheme(
         <span className={styles.icon}>{icon[variant]}</span>
 
         <section className={styles.content}>
-          {heading && <Heading visualLevel="6" children={heading} />}
+          {heading && <h1 className={styles.heading}>{heading}</h1>}
```

Comment:
hmmm why can't we use the Heading and its styles here?

Reply:
The related components - Toast and Banner - use matching styles that weren't available via `Heading`. We also allow for theming those properties in the other components, so I wanted to get this one aligned with their approach.